.github/workflows/build_commits.yaml

```
@@ -0,0 +1,42 @@
+name: build_commits
+
+
+on: [pull_request]
+
+jobs:
+  build_commits:
+    name: build commit code
```

[Useless nit] Since we are not constrained by size, I would suggest something more descriptive, e.g.:
```suggestion
    name: Check if build works for every commit
```

---

I think this job should be named appropriately, please take suggestion by Nicolas into consideration.

---

Sorry for the inconvenience, I missed it.